src/services/inplayer.account.service.ts

```diff
@@ -52,6 +52,34 @@ export const getUser = async (): Promise<Customer> => {
 
 export const getFreshJwtToken = async ({ auth }: { auth: AuthData }) => auth;
 
+export const updateCustomer: UpdateCustomer = async (values) => {
+  try {
+    const consents: { [key: string]: string } = {};
+    values.consents?.map((consent: CustomerConsent) => {
+      if (consent.label) {
+        const { customerId, date, newestVersion, needsUpdate, ...rest } = consent;
+        consents[`consents_${consent.name}`] = JSON.stringify(rest);
```

Comment:
Would it be easier to 'find' the consents if they were in a single field called `customerConsents` that contained the json for all consents? As it currently stands, all of the update calls send all of the consent values already anyhow, so we wouldn't need to worry about partial data and it means less looping and string manipulation because we don't have to read, write, or search for the consent_ names.

Reply:
Certainly if we did individual inserts and updates I would agree, but update consents always takes an array of all consents anyhow.